echopype/convert/api.py

```diff
@@ -430,10 +430,7 @@ def open_raw(
 
     # TODO: the if-else below only works for the AZFP vs EK contrast,
     #  but is brittle since it is abusing params by using it implicitly
-    if SONAR_MODELS[sonar_model]["xml"]:
-        params = xml_chk
-    else:
-        params = "ALL"  # reserved to control if only wants to parse a certain type of datagram
+    params = xml_chk
```

Comment:
I think we can remove these altogether and change the line 441 assignment to be just `params=xml_chk`, and add a comment to say that `xml_chk` is empty if not AZFP files

Reply:
Looking into the code again, it seems that this `params` argument is really only used for passing AZFP XML file. Maybe we should rename it to be something like `file_meta`? (this will go with the `meta_source_filenames` that are populated for AZFP like [here](https://echopype.readthedocs.io/en/latest/data-format-raw.html#asl-environmental-sciences-azfp-data)).

Could you please make this change, and also add a comment at line 492 to say currently this is only used for AZFP XML file?

Reply:
I don't understand this comment, do you mean to rename `params` to `file_meta` in line 433 and populate that value in `_set_convert_params` method?

Reply:
Somehow the line number got messed up, maybe between different commits/branches. I meant this line here: https://github.com/OSOceanAcoustics/echopype/blob/f98e825e808a98dfdf43f66c8be242347d353e3b/echopype/convert/api.py#L444

I meant to say to rename this `params` to `file_meta`, since for AZFP this XML file name eventually becomes `meta_source_filenames` in the set_group functions.

`_set_convert_params` should be removed since we don't use it anymore, but I think we can do that in another PR, so that this one is not too overloaded.

Reply:
Understood, thanks! Will make another PR for removal of `_set_convert_params` after this.

Reply:
Thank you!